lib/ngtsc/module.symbol.ts

```diff
@@ -4,7 +4,7 @@ import type { ComponentSymbol } from './component.symbol';
 import { assertDeps, exists } from './utils';
 
 export class NgModuleSymbol extends Symbol<'NgModule'> {
-  protected readonly annotation = 'NgModule';
+  readonly annotation = 'NgModule';
 
   get deps() {
```

Comment:
If you disagree with my comment above, then this one should also be `protected`.